src/dialog/tooltip/tooltip.component.ts

```diff
@@ -43,4 +43,9 @@ export class Tooltip extends Dialog {
 	onDialogInit() {
 		this.hasContentTemplate = this.dialogConfig.content instanceof TemplateRef;
 	}
+
+	@HostListener("keydown", ["$event"])
+	onBlur(event: KeyboardEvent) {
```

Comment:
Tooltips can have interactive elements, so we can't just close the tooltip onKeydown

Reply:
Yup I meant to make that onBlur